plugins/org.jkiss.dbeaver.ui.ai/src/org/jkiss/dbeaver/ui/ai/engine/openai/OpenAiConfigurator.java

```diff
@@ -270,7 +274,7 @@ public boolean isComplete() {
     }
 
     @Override
-    public Optional<AIEngineProperties> getCurrentProperties() {
+    public Optional<AIEngineProperties> getCurrentProperties() throws DBException {
```

Comment:
This method can't even throw `DBException`, so is `loadSettings` and `saveSettings`.